microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -12,22 +12,26 @@ toc::[]
 
 [id="microshift-4-14-about-this-release"]
 == About this release
+// TODO: Update with the relevant information closer to release.
+{product-title} (link:https://access.redhat.com/errata/RHSA-2023:XXXX[RHSA-2023:XXXX]) is now Generally Available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md[Kubernetes 1.27] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
 
-//The Red Hat build of {product-title} is Technology Preview only. Features and known issues that pertain to {product-title} {ocp-version} are included in this topic. This Technology Preview software is not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using {product-title} in production. Technology Preview provides early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+You can deploy {product-title} clusters to either on-premises, cloud, or disconnected environments.
```

Comment:
This is one of those unfortunate situations where "on-premises" is technically correct from a grammatical standpoint, but "on-premise" has been used incorrectly enough times for it to be the [preferred term per the RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#on-premise): 

```suggestion
You can deploy {product-title} clusters to either on-premise, cloud, or disconnected environments.
```